networker/networker.go

```diff
@@ -149,6 +150,11 @@ type Combined struct {
 	fwmark             uint32
 	mu                 sync.Mutex
 	lanDiscovery       bool
+	// need to memorize route to remote LAN state set on mesh peer connect
+	// according how remote peer has set its permission, for later when
+	// doing mesh refresh which may happen in background e.g. when new peer
+	// was added and NC update message is received
+	routeToRemoteLan bool
```

Comment:
Is this state really necessary? The PreventLocalTraffic function already doesn't return errors on cases when trying to insert the rule while it already exists or trying to delete it when it doesn't exist. So I'm not sure why we need to remember the last state, when we can always just use the current one which is passed by an argument.

Reply:
Ok, I think I got your idea when analyzing network change event handling with refreshVPN and setMesh - all can be handled in `SetupRoutingRules` only need to add extra check to not create duplicate rules

Reply:
Still need to keep state for `refreshVPN` case